src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationTracerHelper.cs

```diff
@@ -56,10 +55,14 @@ public static TracerProviderBuilder UseEnvironmentVariables(this TracerProviderB
     public static TracerProviderBuilder AddSdkAspNetInstrumentation(this TracerProviderBuilder builder)
     {
 #if NET462
-        return builder.AddAspNetInstrumentation();
-#elif NETCOREAPP3_1_OR_GREATER
-        return builder.AddAspNetCoreInstrumentation();
+        builder.AddAspNetInstrumentation();
```

Comment:
Does our existing automatic instrumentation for ASP.NET actually work? It [looks like](https://opentelemetry.io/docs/instrumentation/net/netframework/#aspnet-initialization) the user would have to also modify their `web.config` file for automatic instrumentation to occur. I would also expect us to have the same pattern as AspNetCore where we add the sources here and then actually build the instrumentation later

Reply:
`Web.config` modification is definitely needed, the current asp.net framework instrumentation is covered with integration tests and according to them should work ok. 

The ASP.NET Framework instrumentation actually does not need a detector. Why?
- There are no external types. ASP.NET Framework was built into the core framework. 
- [AspNetInstrumentation.cs](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AspNet/AspNetInstrumentation.cs)
- [HttpInListener.cs](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AspNet/Implementation/HttpInListener.cs)
- [OpenTelemetry.Instrumentation.AspNet.csproj](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AspNet/OpenTelemetry.Instrumentation.AspNet.csproj#L13)

Reply:
Thanks for clarifying, it makes sense that we do not need a detector.

I think what we should do as a new issue is to figure out how to instrument AspNet _without the user changing their web.config_. Currently in our sample application, we don't reference the NuGet package but we do manually reference the HttpModule, which makes the tracing work. Users would ideally not have to touch their web.config to get this working and instead we add the HttpModule automatically for them

Reply:
Yep, that would be the best if possible.